modules.d/01dbus-broker/module-setup.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+# This file is part of dracut.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Prerequisite check(s) for module.
+check() {
+
+    # If the binary(s) requirements are not fulfilled
+    # return 1 to not include the binary.
+    require_binaries busctl || return 1
+    require_binaries dbus-broker || return 1
+    require_binaries dbus-broker-launch || return 1
+
+    # If the module dependency requirements are not fulfilled
+    # return 1 to not include the required module(s).
+    if ! dracut_module_included "systemd"; then
+        derror "dbus-broker needs systemd in the initramfs."
+        return 1
+    fi
+
+    # dbus-broker conflicts with dbus.
+
+    if dracut_module_included "dbus"; then
+        derror "dbus-broker conflicts with dbus in the initramfs."
+        exit 1
+    fi
+
+    # Return 255 to only include the module, if another module requires it.
+    return 255
+
+}
+
+# Module dependency requirements.
+depends() {
+
+    # This module has external dependency on the systemd module.
+    echo systemd
+    # Return 0 to include the dependent systemd module in the initramfs.
+    return 0
+
+}
+
+# Install the required file(s) for the module in the initramfs.
+install() {
+
+    # Create dbus related directories.
+    inst_dir    $dbus
+    inst_dir    $dbusinterfaces
+    inst_dir    $dbusservices
+    inst_dir    $dbussession
+    inst_dir    $dbussystem
+    inst_dir    $dbussystemservices
+    inst_dir    $dbusconfdir
+    inst_dir    $dbusinterfacesconfdir
+    inst_dir    $dbusservicesconfdir
+    inst_dir    $dbussessionconfdir
+    inst_dir    $dbussystemconfdir
+    inst_dir    $dbussystemservicesconfdir
```

Comment:
why so much space after `inst_dir` ?

Reply:
Tab vs Space

Reply:
`.editorconfig` has `indent_style = space`

Reply:
I should have been more clearer so it would not be confused with TAB spaces when I ( or someone else ) presses tab it creates 4 intent spaces, which is what you are seeing there.
```
# Match config files, set indent to spaces with width of four.
[*.sh]
indent_style = space
indent_size = 4 <---
```

The reason for this being there is simply because I think it's cleaner and more readable than having a single space. I can most certainly make a single space between ```inst_$FOO BAR``` if that is preferred and more correct.